src/utils/archive.ts

```diff
@@ -199,7 +200,59 @@ export async function createPublicDrive(
     txs: [driveTx, rootFolderTx]
   };
 }
+export async function sendArDriveFee(
+  keyfile: JWKInterface,
+  arPrice: number,
+  arweave: Arweave
+) {
+  try {
+    // Get the latest ArDrive Community Fee from the Community Smart Contract
+    let fee = arPrice * ((await getArDriveFee()) / 100);
+
+    // If the fee is too small, we assign a minimum
+    if (fee < 0.00001) {
+      fee = 0.00001;
+    }
+
+    // Probabilistically select the PST token holder
+    const holder = await selectTokenHolder();
+
+    // send a fee. You should inform the user about this fee and amount.
```

Comment:
> You should inform the user about this fee and amount.
To whom is this comment directed? If there's nothing for the coder to do here, simply omit this.

Reply:
[Updated](https://github.com/th8ta/ArConnect/pull/33/commits/f76faa99ec9bfb48f340cbe9b4857e805e4e2582)